curp/src/rpc/mod.rs

```diff
@@ -5,13 +5,14 @@ use serde::{de::DeserializeOwned, Deserialize, Serialize};
 
 pub use self::proto::{
     commandpb::{cmd_result::Result as CmdResultInner, CmdResult, ProposeRequest, ProposeResponse},
+    errorpb::Member,
```

Comment:
Why `errorpb`?

Reply:
I have the same question.

Reply:
Modified, previously to fix the proto circular dependency issue, but no longer has this issue